test/fixtures/server-test/webpack.config.js

```diff
@@ -1,12 +1,14 @@
 module.exports = {
+	mode: "development",
+	devtool: false,
 	context: __dirname,
 	entry: "./foo.js",
 	output: {
 		filename: "bundle.js",
 		path: "/"
 	},
 	module: {
-		loaders: [
+		rules: [
```

Comment:
is this change specific to webpack@4?

Reply:
Actually this schema name change started in webpack v2 (`loaders` was deprecated in favor of `rules`), but it wasn’t removed until v4.

Reply:
OK this change can stick around